modules/on-prem/pages/kafka-cluster-setup.adoc

```diff
@@ -1,16 +1,60 @@
-= Set up ML Workbench Enterprise Edition
-
-ML Workbench offers two different communication methods for exporting data from your TigerGraph Database instance to your ML development environment: HTTPS and Kafka streaming.
+= Set up Workbench with Kafka Streaming (Enterprise Only)
+:sectnums:
+:description: Steps to set up ML Workbench with Kafka streaming.
 
+ML Workbench offers two different data transfer protocols for exporting data from your TigerGraph Server instance: HTTPS and Kafka streaming.
 HTTPS data is limited to 2GB of data export at a time, while Kafka streaming does not have this limitation.
-Kafka streaming is only available for the Enterprise Edition.
+Kafka streaming is only available for the Enterprise edition.
+
+NOTE: The ML Workbench supports the `SASL_PLAIN` authentication method for data streaming from Kafka.
+
+The default configurations of this procedure installs Kafka, ML Workbench, as well as TigerGraph Server.
+If you already have instances of these services running, you can remove the corresponding service from the configuration.
+
+== Prerequisites
+* Docker is installed on your machine.
+* Your machine needs to have the following ports open:
+** 2181.
+** 19092.
+** 9092. if you configure Kafka to be listening on another port, then whichever taht port is must be open instead of 9092.
+* You have purchased the ML Workbench Enterprise Edition.
+
+NOTE: This procedure uses `docker-compose` and therefore requires Docker to be installed on your machine.
+If you need to set up Kafka Streaming without using Docker, please open a support ticket.
```

Comment:
You can also set up Kafka directly on your machine without using Docker, or set up a Kafka cluster on your a Kubernetes container management system on any cloud providers. If you need help, please open a support ticket. 

(note: pls feel free to rephrase)